packages/interceptors-opentelemetry/src/workflow/index.ts

```diff
@@ -52,7 +53,13 @@ export class OpenTelemetryInboundInterceptor implements WorkflowInboundCallsInte
   ): Promise<unknown> {
     const context = await extractContextFromHeaders(input.headers);
     const spanName = `${SpanName.WORKFLOW_EXECUTE}${SPAN_DELIMITER}${workflowInfo().workflowType}`;
-    return await instrument(this.tracer, spanName, () => next(input), context);
+    return await instrument(
+      this.tracer,
+      spanName,
+      () => next(input),
+      context,
+      (err) => err instanceof ContinueAsNew
+    );
```

Comment:
Thoughts on functions taking a single object param, versus list of params? I prefer object for readability at call site when it's not super simple to infer the meaning.

Simple: 

```ts
client.getHandle(tripId);
```

("Okay, if getHandle takes an ID, it's probably a workflow ID")

Prefer object:

```ts
new DefaultDataConverter(root) // "What is root?"
new DefaultDataConverter({ protobufRoot: root }) // "Okay it's something for protobufs"

instrument(
  this.tracer,
  spanName,
  () => next(input),
  context,
  (err) => err instanceof ContinueAsNew
) // "What are all these args for? What does returning true from the last callback mean?"
instrument(
  tracer: this.tracer,
  name: spanName,
  fn: () => next(input),
  context,
  acceptableErrors: (err) => err instanceof ContinueAsNew
) // I have a better idea
```

Reply:
I generally agree with you.
In this case I just put this together quickly and didn't put much thought into this.

Because it's mostly internal I don't feel like it's critical.